### PR TITLE
Create codeql.yml

### PR DESCRIPTION
**Pull request by palashgdev**

### CODE REVIEW

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'go', 'javascript', 'python' ]
```
Comment (prashant-shahi) [inline .github/workflows/codeql.yml:35]: I don't think `python` would be required here.

File: .github/workflows/codeql.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "develop", main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "develop" ]
+  schedule:
+    - cron: '21 11 * * 0'
```
Comment (prashant-shahi) [inline .github/workflows/codeql.yml:21]: > 21 11 * * 0

That cron expression translates to "11:21am every Sunday". Any reason for that?

Alternatives would be:
- 10am every sunday: `0 10 * * 0`
- 10am every day: `0 10 * * *`
- 10am every week days: `0 10 * * 1-5`

File: .github/workflows/codeql.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "develop", main ]
```
Comment (prashant-shahi) [inline .github/workflows/codeql.yml:16]: quotes are optional.

Also, we should keep it consistent: either use quotes or not.

**Review by prashant-shahi — CHANGES_REQUESTED** [review]

**Review by prashant-shahi — APPROVED** [review]
Workflow changes look good..

Check with @pranshuchittora for frontend related changes.

**Review by pranshuchittora — APPROVED** [review]

**Comment by pranshuchittora** [comment]
Pls, resolve the merge conflicts. WIll re-review after that

**Pull request closed without merging**